### PR TITLE
Added Apple M1 Arm64 to the PR and Release flow

### PR DESCRIPTION
**Pull request by tkmcmaster**
- [Added Apple M1 Arm64 to the PR and Release flow](https://github.com/FamilySearch/pewpew/pull/174/commits/6383859b3db80f35f257f44f2d51f6c79fb9a76e)

### CODE REVIEW

**Review by burnsidepj — APPROVED** [review]

**Comment by summersjc** [comment]
This is actually not correct

**Review by summersjc — CHANGES_REQUESTED** [review]
I was wrong before, this should be called something different.

https://rust-lang.github.io/rustup-components-history/aarch64-apple-darwin.html

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -175,6 +180,13 @@ jobs:
           echo "ASSET_NAME=$asset_name" >> $GITHUB_ENV
           XZ_OPT=-9 tar -C ./target/release/ -cJf $asset_name pewpew
 
+      - name: Compress for macOS
+        if: matrix.os == 'macos-latest-xlarge'
+        run: |
+          asset_name="pewpew-$VERSION-apple-darwin-arm64.tar.xz"
```
Comment (summersjc) [inline .github/workflows/release.yml:186]: I think we should name the binary aarch64-apple-darwin

**Review by summersjc — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (tkmcmaster) [reply .github/workflows/release.yml]: @summersjc Done

**Review by summersjc — APPROVED** [review]

**Pull request merged**